### ISSUE TITLE
Duplicate phenotypes?

### ISSUE THREAD

**Issue by Upliner** [animate1978/MB-Lab#299]
For example, amazonid and south_american are practically distinguishable

`
--- amazonid.json       2020-08-08 01:56:10.665254935 +0300
+++ south_american.json 2020-05-28 22:04:40.854955048 +0300
@@ -6,6 +6,7 @@
     "Waist_Size": 0.68,
     "Torso_Length": 0.95,
     "Head_Size": 0.65,
+    "Body_Size": 0.5,
     "Legs_UpperlegLength": 0.23,
     "Mouth_SizeX": 0.68,
     "Mouth_LowerlipVolume": 0.9,
@@ -15,7 +16,7 @@
     "Eyelids_Angle": 0.45,
     "Mouth_UpperlipVolume": 0.65,
     "Nose_Curve": 0.53,
-    "Mouth_UpperlipSizeZ": 0.47,
+    "Mouth_UpperlipSizeZ": 0.475,
     "Eyelids_Crease": 0.9,
     "Nose_TipPosZ": 0.38,
     "Chin_Prominence": 0.4,
`

**Comment by animate1978** [comment]
Yes @Upliner I forgot to remove some of the phenotypes during release.

I will be going through and fixing these
